modules/distribution-service-aws-s3/src/main/java/org/opencastproject/distribution/aws/s3/AwsS3DistributionServiceImpl.java

```diff
@@ -921,19 +921,12 @@ protected String process(Job job) throws Exception {
           return (retractedElements != null) ? MediaPackageElementParser.getArrayAsXml(Arrays.asList(retractedElements))
                   : null;
         /*
-         * TODO
-         * Commented out due to changes in the way the element IDs are passed (ie, a list rather than individual ones
-         * per job). This code is still useful long term, but I don't have time to write the necessary wrapper code
-         * around it right now.
-         * case Restore:
-         * String fileName = arguments.get(3);
-         * MediaPackageElement restoredElement = null;
-         * if (StringUtils.isNotBlank(fileName)) {
-         * restoredElement = restoreElement(channelId, mediaPackage, elementIds, fileName);
-         * } else {
-         * restoredElement = restoreElement(channelId, mediaPackage, elementIds, null);
-         * }
-         * return (restoredElement != null) ? MediaPackageElementParser.getAsXml(restoredElement) : null;
+         * TODO Commented out due to changes in the way the element IDs are passed (ie, a list rather than individual
+         * ones per job). This code is still useful long term, but I don't have time to write the necessary wrapper code
+         * around it right now. case Restore: String fileName = arguments.get(3); MediaPackageElement restoredElement =
+         * null; if (StringUtils.isNotBlank(fileName)) { restoredElement = restoreElement(channelId, mediaPackage,
+         * elementIds, fileName); } else { restoredElement = restoreElement(channelId, mediaPackage, elementIds, null);
+         * } return (restoredElement != null) ? MediaPackageElementParser.getAsXml(restoredElement) : null;
```

Comment:
I'm not too happy about commented code in general (though that's not your fault), but I feel like this makes this snippet very hard to read. I don't think the space won by this change is worth it.

Reply:
This change was unintentional (as well as others that changed the original formatting). I will revert all of them so that you only see what really changed.

Reply:
Actually, most of the formatting changes - except for this one - seemed quite reasonable. Maybe put them in a separate commit? :)

Reply:
Hi @KatrinIhler , I don't think this is related at all with this PR?

Reply:
it's not, but since you already did the work - you could also file a separate PR? I'd approve it. :D

Reply:
Ok, separate PR then. No problem!